README.rst

```diff
@@ -1,83 +1,161 @@
-- MXNet: https://sagemaker.readthedocs.io/en/stable/using_mxnet.html
-- PyTorch: https://github.com/aws/sagemaker-python-sdk#pytorch-sagemaker-estimators
-- Chainer: https://github.com/aws/sagemaker-python-sdk#chainer-sagemaker-estimators
+Building the container
+~~~~~~~~~~~~~~~~~~~~~~
+
+We can build the container like any docker container:
+
+.. code:: shell
+
+   docker build -t tf-2.0 .
+
+.. _header-n971:
+
+Training with Local Mode
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+We can use `Local
+Mode <https://sagemaker.readthedocs.io/en/stable/overview.html#local-mode>`__
+to test the container locally:
+
+.. code:: python
+
+   from sagemaker.estimator import Estimator
+
+   estimator = Estimator(image_name='tf-2.0',
+                         role='SageMakerRole',
+                         train_instance_count=1,
+                         train_instance_type='local')
+
+   estimator.fit()
+
+To see a complete example on how to create a container using SageMaker
+Container, including pushing it to ECR, see `Building your own
+container <https://github.com/awslabs/amazon-sagemaker-examples/blob/master/advanced_functionality/tensorflow_bring_your_own/tensorflow_bring_your_own.ipynb>`__.
+
+.. _header-n975:
 
 How a script is executed inside the container
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+---------------------------------------------
+
+The environment variable **SAGEMAKER_PROGRAM** determines the entrypoint
+name that will be executed during training. This script must be located
+under the folder **/opt/ml/model**. The following scripts are supported:
+
+-  **Python scripts**: uses the Python interpreter for any script with
+   **.py** suffix
+
+-  **Shell scripts**: uses the Shell interpreter to execute any other
+   script
+
+When training starts, the interpreter executes the entrypoint, from the
+example above:
+
+``python train.py``
+
+.. _header-n984:
+
+Mapping hyperparameters to script arguments
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Any hyperparameters provided by the training job will be passed by the
+interpreter to the entry point as script arguments. For example the
+training job:
 
-When the container starts for training, **SageMaker Containers**
-installs the user script as a Python module. The module name matches the
-script name. In the case above, **user-script.py** is transformed in a
-Python module named **user-script**.
+.. code:: python
 
-After that, the Python interpreter executes the user module, passing
-``hyperparameters`` as script arguments. The example above will be
-executed by **SageMaker Containers** as follow:
+   from sagemaker.chainer import Chainer
 
-.. code:: bash
+   estimator = Chainer(entry_point='user_script.sh',
+                       hyperparameters={'batch-size':256,
+                                        'learning-rate':0.0001,
+                                        'communicator':'pure_nccl'},
+                       ...)
```

Comment:
not sure if it's worth tying this part to the Python SDK

Reply:
I think it'd be clearer to show the hyperparameters as JSON - less other code to serve as a distraction